services/portainer/docker-compose.yml

```diff
@@ -35,6 +42,13 @@ services:
     deploy:
       mode: replicated
       replicas: 1
+      resources:
+        limits:
+          cpus: "1"
+          memory: 250M
```

Comment:
I think this one might benefit form higher limits. Sometimes it's very slow. I would have 2 cpus for it.

Reply:
I made it 2. I assume that it is slow due to other reasons